bin/console

```diff
@@ -0,0 +1,8 @@
+#!/usr/bin/env ruby
+
+lib = File.expand_path(File.join('..', 'lib'), __dir__)
+$LOAD_PATH.unshift(lib) unless $LOAD_PATH.include?(lib)
+require 'zip'
```

Comment:
I think it's possible to do this with `require_relative` now (I think since 1.9), without the path manipulation:
```rb
require_relative '../lib/zip'
```

Reply:
I always forget about `require_relative`. I'll change that.

Reply:
Having now tried this, `require_relative` doesn't actually work, because everything in `lib/zip.rb` is `require`d, so also relies on `$LOAD_PATH` being set correctly.

So I could fix this, but it would be a lot more involved that just changing this file. Happy to do so if you think it's worth it?